pkgs/top-level/python-packages.nix

```
@@ -748,6 +748,8 @@ self: super: with self; {
 
   async-timeout = callPackage ../development/python-modules/async_timeout { };
 
+  async-tkinker-loop = callPackage ../development/python-modules/async-tkinker-loop { };
```

```suggestion
  async-tkinter-loop = callPackage ../development/python-modules/async-tkinter-loop { };
```
This also needs to be changed.